schemas/styles/csl.rnc

```diff
@@ -649,6 +650,72 @@ div {
 }
 # ==============================================================================
 
+## cs:title Rendering Element
```

Comment:
So this is a new rendering element? 

Two things:

1. is it _really_ necessary; this can't be done adequately with attributes on cs:text?
2. not sure if this is a technical problem or not, but will note this is the same element name as `cs:info/cs:title`.

Reply:
ad 1: No, it's not necessary. We could also allow children on `cs:text`. Or it could also work with attributes on `cs:text`, but I think that would be way to complex. You'd need to duplicate every possible attribute with a `subtitle` prefix. 
But I'm not sure which one is a bigger change.
2. I don't know, but we could just change it to `cs:titles`? Would be also more appropriate as this will render different titles (`container-title`, `volume-title`, etc.)

Reply:
On your edit: Could work, but it's not ideal as you won't have access to the original delimiters here. And some styles (APA) require that.

Reply:
I adjusted the formatting of your main comment at top to make this more clear.

From my understanding, 1, 3, and 4 "requirements" can be addressed without new elements. Is that correct?

It is only 2 that adds "complexity"?

Reply:
> On your edit: Could work, but it's not ideal as you won't have access to the original delimiters here. And some styles (APA) require that.

I'm not following here.

Reply:
Some styles normalize delimiters, other do not. APA for example will not normalize delimiters, i.e. it will render titles with whichever punctuation was there in the first place. In your solution with a group you'll only be able to use the delimiter specified on `cs:group`, but not the original delimiters.

```
<group delimiter=": ">
  <text variable='title-main" font-weight="bold"/>
  <text variable='title-sub"/>
</group>
```

Reply:
That's helpful. 

But the language you use to explain that sounds like it could be a global style option?

Hoping @adam3smith and @rmzelle can find time to look at this too.